web/satellite/src/views/forgotPassword/ForgotPassword.vue

```diff
@@ -46,10 +47,12 @@
                     this.$store.dispatch(NOTIFICATION_ACTIONS.SUCCESS, 'Please look for instructions at your email');
                 },
                 onBackToLoginClick: function() {
+                    this.$segment.track(EVENTS.CLICKED_BACK_TO_LOGIN);
                     this.$router.push(ROUTES.LOGIN.path);
                 },
                 onLogoClick: function () {
                    location.reload();
+                   this.$segment.track(EVENTS.CLICKED_LOGO);
```

Comment:
may be it is better to track the event before page is reloaded? If it works than its fine. It just that it is more clear that way in my opinion.

Reply:
That's a good point